dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Basics/BooleanTable.mo

```diff
@@ -0,0 +1,49 @@
+within Dynawo.Electrical.Controls.Basics;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model BooleanTable "Generates a Boolean output signal based on a vector of time instants"
```

Comment:
Maybe say here that the table is given by a file

Reply:
Done